onadata/apps/api/management/commands/delete_users.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = 'Delete users'
+
+    def add_arguments(self, parser):
+        parser.add_argument('user', nargs='*')
+
+    def handle(self, *args, **kwargs):
+        users = kwargs['user']
+
+        if users:
+            for user in users:
+                user_details = user.split('-')
```

Comment:
Can you add more docstrings to show how the command can be used? Looks like you provide a list of users that are dash (`-`) separated, which I think might not work especially where we have a username or email with a dash in it, dashes in emails and usernames are quite common.